tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html

```diff
@@ -25,7 +25,8 @@
 <link rel="import" href="../paper-progress/paper-progress.html">
 <link rel="import" href="../paper-spinner/paper-spinner-lite.html">
 <link rel="import" href="../paper-toast/paper-toast.html">
-<link rel="import" href="../tf-imports/polymer.html">
+<link rel="import" href="../polymer/polymer.html">
```

Comment:
Why is this polymer import changing?  Is this a merge issue?

Reply:
Yes. Fixed the merge issue.